ci: update dependabot to focus on security

DaniAkash opened the pull request:
Added open-pull-requests-limit, enabled beta ecosystems (for bun support) and only allow only security updates

felarof99 commented on src/lib/browser/ElementFormatter.ts:
```diff
@@ -55,16 +55,16 @@ export class ElementFormatter {
    */
   formatElement(node: InteractiveNode): string {
     let SHOW_INDENTATION = true;
```
Why aren't others `const` here?

shivammittal274 replied:
because in IF condition, values are changing

felarof99 reviewed (approved):
I had one minor nit. Greptile has few more nits. Let's address those first and then merge the PR. I'm approving.

DaniAkash committed:
ci: update dependabot to focus on security

Added open-pull-requests-limit, enabled beta ecosystems (for bun support) and only allow only security updates

DaniAkash committed:
chore: fix whitespaces

DaniAkash committed:
ci: update dependency groups to only apply to security-updates
